spec/support/shared_examples.rb

```diff
@@ -0,0 +1,35 @@
+shared_examples 'a patcher' do
+  let(:patcher) { described_class.new(patch_location) }
+  let(:patch_location) { nil }
```

Comment:
Possibly we could lose this and (see comment below for [line 12](https://github.com/alphagov/dfid-transition/pull/7/files#diff-1313a61b0b6e102015ddddd7444ebe53R12)) ...